stylesheet/main.css

```diff
@@ -75,11 +90,13 @@ footer {
     box-shadow: 0 0 4px rgba(0,0,0,0.2);
 }
 
+.mugshot {
+    float: right;
+    position: relative;
+    top:-1em;
+}
 .mugshot img {
-     float: right;
-     top:-1em;
     box-shadow: 2px 2px 3px rgba(0,0,0,0.5);
-    background-image:url("paperclip.png")；
 }

```

Comment:
The float and position should apply to the .mugshot element (the `<figure>`) and not the `<img>`
The background image should apply to .paperclip